app/src/views/GrantRoundDetails.vue

```diff
@@ -144,11 +151,12 @@ import {
   hasStatus,
 } from 'src/utils/utils';
 // --- Types ---
-import { GrantRound } from '@dgrants/types';
+import { Grant, GrantRound } from '@dgrants/types';
 import { GrantRound as GrantRoundContract } from '@dgrants/contracts';
 
 function useGrantRoundDetail() {
-  const { grantRounds, poll } = useDataStore();
+  const { grantRounds, grantRoundMetadata: metadata, grants: allGrants, grantMetadata, poll } = useDataStore();
```

Comment:
Not a fan of the variable name `metadata` since it's not clear whether it's for a round or a grant. Perhaps just stick with the original name?

Reply:
Ah I see, you redefine `grantRoundMetadata` as a computed property. Maybe `_grantRoundMetadata` then.

But I don't think this and `grants` need to be computed properties. I think you can just set them on mount with `onMounted`, because (I believe) each time you change the URL param to a round the component is destroyed the mounted

Reply:
I've renamed it to `_grantRoundMetadata` for now ! Not sure how do it via `onMounted` 
Any chance you could show me how to go about that ?

Reply:
@thelostone-mc and I realized computed makes more sense here, so marking this resolved